### PR TITLE
GadgetsRevived.GadgetsRevived version 1.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/100053)

### CODE REVIEW

File: manifests/g/GadgetsRevived/GadgetsRevived/1.0/GadgetsRevived.GadgetsRevived.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: GadgetsRevived.GadgetsRevived
+PackageVersion: 1.0
+PackageLocale: en-US
+Publisher: GadgetsRevived
+PackageName: Gadgets Revivied
```
Comment (Trenly) [inline manifests/g/GadgetsRevived/GadgetsRevived/1.0/GadgetsRevived.GadgetsRevived.locale.en-US.yaml:8]:
```suggestion
PackageName: Windows Desktop Gadgets
```
Should Match Control Panel

File: manifests/g/GadgetsRevived/GadgetsRevived/1.0/GadgetsRevived.GadgetsRevived.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: GadgetsRevived.GadgetsRevived
+PackageVersion: 1.0
```
Comment (Trenly) [inline manifests/g/GadgetsRevived/GadgetsRevived/1.0/GadgetsRevived.GadgetsRevived.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.0
```
Should match control panel. Version folder will need to be updated when changed

File: manifests/g/GadgetsRevived/GadgetsRevived/1.0/GadgetsRevived.GadgetsRevived.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: GadgetsRevived.GadgetsRevived
+PackageVersion: 1.0
```
Comment (Trenly) [inline manifests/g/GadgetsRevived/GadgetsRevived/1.0/GadgetsRevived.GadgetsRevived.yaml:5]:
```suggestion
PackageVersion: 2.0
```
Should match control panel. Version folder will need to be updated when changed

File: manifests/g/GadgetsRevived/GadgetsRevived/1.0/GadgetsRevived.GadgetsRevived.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: GadgetsRevived.GadgetsRevived
+PackageVersion: 1.0
+PackageLocale: en-US
+Publisher: GadgetsRevived
+PackageName: Gadgets Revivied
+License: Propeiry
```
Comment (Trenly) [inline manifests/g/GadgetsRevived/GadgetsRevived/1.0/GadgetsRevived.GadgetsRevived.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Manifest validation error:
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\g\GadgetsRevived\GadgetsRevived\1.0', expected 'manifests\g\GadgetsRevived\GadgetsRevived\2.0'

These are in the `\1.0` folder but need to be in the `\2.0` folder.

**Comment by Trenly** [comment]
Close with reason: Stale PR;

**Pull request closed without merging**